README.md

```diff
@@ -1,16 +1,49 @@
-# Title
+# JavaScript Fundamentals: Scope Chain Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+1.  Identify the difference between const and let
+2.  Identify how functions determine scope in Javascript
 
 ## Introduction
 
-## SWBAT 1
+Scuber's drivers are beginning to take some notes down about their customers.
+Use your knowledge of scope to access and change some of these variables.
 
-## SWBAT 2
+## Instructions
 
-## Conclusion
+A word of warning: In this lab, you will practice doing some bad things. For
+example, we'll ask you to declare variables in global scope, and write functions
+that break down. We do this because, by testing limits and exposing the inner
+workings of a language, you can better understand it, and better know what to
+avoid in the future.
 
-## Resources
+Ok, these are your tasks:
+
+- customerName: Declare a variable in global scope called customerName using the
+  `var` keyword.
+
+- upperCaseCustomerName: Write a function that accesses that global customerName
+  variable, and uppercases it.
+
+- setBestCustomer: Write a function that when called, declares a variable called
+  `bestCustomer` in global scope and assigns it to be `'not bob'`. (Poor bob.)
```

Comment:
The tone of this lab is a little strange (If I recall correctly this is an adaptation of an old lab), can we change bestCustomer to be something along the lines of "mostFrequentCustomer", change all other instances accordingly.